server/src/modules/users/users.module.ts

```diff
@@ -15,6 +15,6 @@ import { User, UserSchema } from '../../mongo/schemas/user.schema';
   ],
   controllers: [UsersController],
   providers: [UsersService],
-  exports: [UsersService],
+  exports: [UsersService, MongooseModule],
```

Comment:
Why is it exporting mongoose module?

Reply:
I get this without:

![image](https://github.com/user-attachments/assets/1c4ccfa9-5bc5-4df4-9113-d480c6e2bb1e)

Reply:
no module should export MongooseModule - it should be only imported